src/quo2/components/info/page_nav.cljs

```diff
@@ -0,0 +1,242 @@
+(ns quo2.components.info.page-nav
+  (:require [quo.react-native :as rn]
+            [quo2.foundations.colors :as colors]
+            [quo2.components.icon :as icons]
+            [status-im.utils.dimensions :as dimensions]
+            [clojure.string :as string]
+            [quo2.components.markdown.text :as text]))
+
+(def ^:private centrify
+  {:style {:display :flex
+           :justify-content :center
+           :align-items     :center}})
+
+(def ^:private align-left
+  (update-in
```

Comment:
its really hard to read, could you pls do one line ?

Reply:
Done